charts/opensearch/README.md

```diff
@@ -128,6 +128,10 @@ helm uninstall my-release
 | `serviceMonitor.enabled` | Enables the creation of a [ServiceMonitor] resource for Prometheus monitoring. Requires the Prometheus Operator to be installed in your Kubernetes cluster. | `false` |
 | `serviceMonitor.path` | Path where metrics are exposed. Applicable only if `serviceMonitor.enabled` is set to `true`. | `/_prometheus/metrics` |
 | `serviceMonitor.interval` | Interval at which metrics should be scraped by Prometheus. Applicable only if `serviceMonitor.enabled` is set to `true`. | `10s` |
+| `serviceMonitor.basicAuth.enabled`        | Wheter or not the serviceMonitor should use basic auth                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | `false`                                                                                                         |
+| `serviceMonitor.basicAuth.existingSecret` | When using basicAuth for the serviceMonitory, use an existing secret                                                                                                                                                                                                                                                                                                                                                                                                                                                               | `""`                                                                                                            |
```

Comment:
Is `serviceMonitory` typo?

Reply:
Hey @eyenx please take a look, we can get this merged.
Thanks

Reply:
@prudhvigodithi fixed.